spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/ManifestFileBean.java

```diff
@@ -36,6 +36,7 @@ public class ManifestFileBean implements ManifestFile, Serializable {
   private Long addedSnapshotId = null;
   private Integer content = null;
   private Long sequenceNumber = null;
+  private Long firstRowId = null;
```

Comment:
Ok this may not be quite right, forgot about delete manifests...

Reply:
Why would this not work for delete manifests?

Reply:
It should (it'd just be null for delete manifests). I confused myself when debugging an issue with failing tests but those tests are unrelated to if it's a delete manifest or not. Checkout my comment below on why I removed the getter on my latest update